ghost-to-markdown/src/index.js

```diff
@@ -0,0 +1,211 @@
+import 'dotenv/config';
+import fs from 'fs';
+import { join } from 'path';
+import render from 'mustache';
+import GhostAdminAPI from '@tryghost/admin-api';
+import yargs from 'yargs';
+import { hideBin } from 'yargs/helpers';
+import { fileURLToPath } from 'url';
+import { dirname } from 'path';
+import winston from 'winston';
+
+import MarkdownRendererFactory from '../lib/index.js';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+// Configure Winston logger
+const logger = winston.createLogger({
+  level: 'info',
+  format: winston.format.combine(
+    winston.format.colorize(),
+    winston.format.timestamp(),
+    winston.format.printf(({ timestamp, level, message }) => {
+      return `${timestamp} ${level}: ${message}`;
+    })
+  ),
+  transports: [
+    new winston.transports.Console(),
+    new winston.transports.File({ filename: 'combined.log' })
+  ]
+});
+
+const wait = (seconds) =>
+  new Promise((resolve) => setTimeout(resolve, seconds * 1000));
+
+const apiUrl = process.env.GHOST_API_URL;
+const apiKey = process.env.GHOST_API_KEY;
+const apiVersion = 'v3.39';
+
+if (!apiUrl || !apiKey) {
+  logger.error(
+    'GHOST_API_URL and GHOST_API_KEY must be set in the environment variables.'
+  );
+  process.exit(1);
+}
+
+const api = new GhostAdminAPI({
+  url: apiUrl,
+  key: apiKey,
+  version: apiVersion
+});
+
+const figureTemplate = `
+<figure>
+  <img src="{{Src}}">
+  {{#Caption}}<figcaption>{{Caption}}</figcaption>{{/Caption}}
+</figure>
+`;
```

Comment:
I think this is a good way to handle the images in Ghost, since all of them that are added in the rich text editor are `img` elements wrapped in `figure` elements, with an optional `figcaption`.

The mobile doc format seems to have an `alt` field for images if one was added by the author. Is it possible to add the `alt` attribute to images with alt text?

Reply:
Since we're handling images from the mobiledoc, do we need this part?

Reply:
No, I don't think we need this anymore. We still have a flag for this, but returning images as figures is for a responsive image feature from Ghost, but only if authors wrote using the rich text editor.

Like you mentioned, since we're using the mobiledoc version of the article are converting them into markdown, we can probably remove all this figure related stuff.